app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
```

Comment:
- [ ] This module requires that `User` model has method that returns 3 most recent posts. This should be similar to the example below:

```ruby
# app/models/user.rb
...

def recent_post
  posts.order('created_at Desc').limit(3)
end
```

Reply:
Done